include/framebuffer.h

```diff
@@ -0,0 +1,26 @@
+// clang-format Language: C
+#ifndef FRAMEBUFFER_H
+#define FRAMEBUFFER_H
+
+#include "mib_header_DAC.h"
+#include "mib_header_MQ1.h"
+#include "mib_macros.h"
+#include "mib_utils.h"
+#include "mq1_quad.h"
+#include "mq1_single.h"
+#include "read_mq1_headers.h"
+#include <stdint.h>
```

Comment:
I believe the header file names needs to be changed now. Same for all other files.